pkg/cgroups/systemd_linux.go

```diff
@@ -260,3 +262,35 @@ func resourcesToProps(res *configs.Resources, v2 bool) (map[string]uint64, map[s
 
 	return uMap, sMap, bMap, iMap, structMap
 }
+
+func RangeToBits(str string) ([]byte) {
+	bits := new(big.Int)
+
+	for _, r := range strings.Split(str, ",") {
+		// allow extra spaces around
+		r = strings.TrimSpace(r)
+		// allow empty elements (extra commas)
+		if r == "" {
+			continue
+		}
+		startr, endr, ok := strings.Cut(r, "-")
+		if ok {
+			start, _ := strconv.ParseUint(startr, 10, 32)
+			end, _ := strconv.ParseUint(endr, 10, 32)
```

Comment:
please propagate these errors, if they happen

Reply:
I added error handling. However, in Podman, values are checked in advance, so no errors occur.
I also added a test to check the value of the converted cpuset.